### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by OLIPLICHE**
# Bookstore

- I add styling. 
- I use the Zeplin UI library to design the website.
- I add Linters Config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # BookStore
 
-I lay foundations for Bookstore website. I create a React and Redux app. I structure the files using the "feature folder" approach. Also I set up routing using React Router, I configure the Redux Store, the reducer for adding and removing books. My application is connected to an existing API.
+I lay foundations for Bookstore website. I create a React and Redux app. I structure the files using the "feature folder" approach. Also I set up routing using React Router, I configure the Redux Store, the reducer for adding and removing books. My application is connected to an existing API. Thank you to Zeplin for the wonderful UI provided.  
+
```
Comment (cyonii) [inline README.md]: - [x] Your project description should be a human-readable description, it appears as though you have written it in imperative mood as you would, git commit messages. You can use [Grammarly](https://grammarly.com) to eliminate common errors in writing.

**Review by cyonii — CHANGES_REQUESTED** [review]
Hi @OLIPLICHE, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review to see the changes you need to make._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] [ OPTIONAL ] Rather than disabling eslint guards as seen below, you can simply install the missing dependencies:
```js
// Remove
/* eslint-disable import/no-extraneous-dependencies */
```
and run the command below to install the missing dependencies.
```bash
npm i -S uuid
```
```bash
npm i -S prop-types
```



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddNewBook.js [hunk 2]
```diff
@@ -30,25 +30,33 @@ const AddNewBook = ({ addBookToStoreProps }) => {
...
-        onChange={onChange}
-        value={formInputs.category}
-        required
-      />
-      <button type="button" onClick={addBook}>Add New Book</button>
-    </form>
+    <>
+      <p className="add-form-header">ADD NEW BOOK</p>
+      <form className="add-form">
+        <div className="add-form__left">
```
Comment (cyonii) [inline src/components/AddNewBook.js:36]: - [x] Please make sure your project UI matches the one provided on Zeplin. Please compare the images and make the necessary changes
Screen Shot 2021-10-09 at 4 07 54 PM
Screen Shot 2021-10-09 at 4 08 21 PM

**Review by afanimartin — APPROVED** [review]
Hi @OLIPLICHE, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**